website/playground/sidebar/options.js

```diff
@@ -51,3 +54,10 @@ export default function(props) {
       throw new Error("unsupported type");
   }
 }
+
+function getDescription(option) {
+  const description = !option.inverted
```

Comment:
Can you invert this ternary so we don’t have a negated condition? (i.e. `inverted ? oppositeDescription : description`)